src/main/java/com/google/sps/util/SearchUtils.java

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.util;
+
+import com.google.appengine.api.datastore.Query;
+import java.lang.UnsupportedOperationException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * A utility class with static methods to complete search related functions.
+ */
+public class SearchUtils {
+  /**
+   * Takes a string and returns a standard version of the string depending on the implementation.
+   * This is used for processing a string before parsing for different query data.
+   * Current implementation returns strings trimmed and in lowercase.
+   */
+  public static String standardize(String str) {
+   String standard = str.trim();
+   standard = standard.toLowerCase();
+   return standard;
```

Comment:
could you simplify this to `return str.trim().toLowerCase();`?

Reply:
I could, but I'm not sure if any problems or changes will be made in the future that will require the standardizing method to be more comprehensive, and for that reason I decided to leave it split. I could simplify it to `String standard = str.trim().toLowerCase()` and then `return standard;` if you think that's better

Reply:
At this point the logic is pretty simple, so I don't think you'd run into serious issues if you needed to add more to this method in the future. What you suggested looks good to me, but it's up to you - I'd suggest choosing whichever one you think is the most readable.

Reply:
Sounds good